packs/feats/divine-rebuttal.json

```diff
@@ -25,6 +25,10 @@
             "title": "Pathfinder Player Core"
         },
         "rules": [],
+        "selfEffect": {
+            "name": "Effect: Divine Rebuttal",
+            "uuid": "Compendium.pf2e.feat-effects.Item.Effect: Divine Rebuttal"
+        },
```

Comment:
Since this is instantaneous it should just be a toggle on this item, then the effect can just be for allies